### PR TITLE
Add Go runtime GroupUpdate userID param

### PR DESCRIPTION
**Pull request by sesposito**
nakama-common PR: https://github.com/heroiclabs/nakama-common/pull/117

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -75,3 +75,5 @@ require (
 	golang.org/x/sys v0.0.0-20220610221304-9f5ed59c137d // indirect
 	golang.org/x/text v0.3.7 // indirect
 )
+
+replace github.com/heroiclabs/nakama-common => ../nakama-common
```
Comment (tomglenn) [inline go.mod:79]: Is this change intentional?

Comment (sesposito) [reply go.mod:79]: discussed elsewhere
